#!/usr/bin/env python
PACKAGE = "prizm_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("drive_speed",              double_t, 0, "Drive Speed", 0.0, 0.0, 1.0)
gen.add("steering_multip",          double_t, 0, "Steering multip", 1.0, 0.0, 5.0)

exit(gen.generate(PACKAGE, "prizm_control", "MoveToObject"))
